src/heap/code-range.cc

```diff
@@ -169,8 +170,15 @@ bool CodeRange::InitReservation(v8::PageAllocator* page_allocator,
   VirtualMemoryCage::ReservationParams params;
   params.page_allocator = page_allocator;
   params.reservation_size = requested;
+#ifdef V8_COMPRESS_POINTERS_IN_MULTIPLE_CAGES
+  // To share code cage between two isolate groups
+  // we have to use 4GB alignment to make addresses
+  // to code cage base independent.
```

Comment:
Reflow comment.  I guess I would reword: "to avoid the need to relocate offsets into the code cage when cloning an isolate group, we align the code cage on a 4 GB boundary."

which, though, I am not sure why that is; do we sometimes compress a code cage pointer by truncation instead of subtracting off the base address?  Why does non-4GB alignment work for the stock configuration but it would not work for us?